src/dom/dom.js

```diff
@@ -772,11 +772,14 @@ p5.prototype.createSelect = function() {
  * assign methods of <a href="#/p5.Element/">p5.Element</a>.
  * - `.option(value, [label])` can be used to create a new option for the
  *   element. If an option with a value already exists, it will be returned.
+ *   It is recommended to use string values as input for `value`.
  *   Optionally, a label can be provided as second argument for the option.
- * - `.remove(value)` can be used to remove an option for the element.
+ * - `.remove(value)` can be used to remove an option for the element. String
+ *   literals recommended as input for `value`.
  * - `.value()` method will return the currently selected value.
  * - `.selected()` method will return the currently selected input element.
- * - `.selected(value)` method will select the option and return it.
+ * - `.selected(value)` method will select the option and return it. String
```

Comment:
For here and for `.remove(value)` above, string literals is not the recommendation but rather just strings as string literals have a specific meaning of only including strings defined inline with quotemarks and not string variables.

Reply:
Thanks for the check! I have changed "string literals" to "string values".